spec/features/record_phone_call_activity_spec.rb

```diff
@@ -31,53 +31,33 @@
     expect_to_be_on_record_phone_call_page
 
     # Test required fields
-    click_button "Continue"
+    click_on "Add phone call"
 
     expect(page).to have_error_messages
-    expect(page).to have_error_summary "Correspondence date cannot be blank"
+    expect(page).to have_error_summary "Enter the date of call"
 
     # Test date validation
     fill_in "Day", with: "333"
-    click_button "Continue"
+    click_on "Add phone call"
 
     expect(page).to have_error_messages
-    expect(page).to have_error_summary "Correspondence date must include a month and year"
+    expect(page).to have_error_summary "Date of call must include a month and year"
 
     fill_in_record_phone_call_form(name: name, phone: phone, date: date)
 
     expect_to_be_on_record_phone_call_details_page
 
     # Test required fields
-    click_button "Continue"
+    click_button "Add phone call"
 
     expect(page).to have_error_summary "Please provide either a transcript or complete the summary and notes fields"
 
-    attach_file "correspondence_phone_call[transcript][file]", file
-    click_button "Continue"
-
-    expect_to_be_on_confirm_phone_call_details_page
-    expect_confirm_phone_call_details_page_to_show_entered_information(name: name, phone: phone, date: date, file: file)
-
-    # Test edit details pages retain entered information
-    click_link "Edit details"
-
-    expect_to_be_on_record_phone_call_page
-    expect_record_phone_call_form_to_have_entered_information(name: name, phone: phone, date: date)
-
-    click_button "Continue"
-
-    expect_to_be_on_record_phone_call_details_page
-
-    within_fieldset "Details" do
-      expect(page).to have_css("a", text: File.basename(file))
-    end
-
-    click_button "Continue"
+    attach_file "phone_call_correspondence_form[transcript]", file
```

Comment:
You can use the label text as a selector here:

```suggestion
    attach_file "Upload a file", file
```